### ISSUE TITLE
Metabolic pathway data

### ISSUE THREAD

**Issue by karafecho** [NCATSTranslator/Feedback#64]
The QotM identified PharmGKB](https://www.pharmgkb.org/), [RaMP DB](https://rampdb.nih.gov/), [Rhea-DB](https://www.rhea-db.org/), and [KEGG Pathway DB](https://www.genome.jp/kegg/pathway.html) as potential sources of metabolic pathway data.

**Comment by marcdubybroad** [comment]
@karafecho The Genetics KP team is in the process of promoting new data through our stack that has KEGG and WIKIPATHWAYS data. It's currently in our dev stack.
We've only loaded disease/pathway association data where the p-value is less than 0.05.
The p-values are calculated from the Magma process aggregation of the genes' p-values in the pathway for that disease/phenotype.
Example query for our dev server (https://translator.broadinstitute.org/genetics_provider/trapi/v1.3/query)
```
{
    "message": {
      "query_graph": {
        "nodes": {
          "n0": {
            "ids": ["KEGG:hsa05322"]
          },
          "n1": {
            "categories": ["biolink:NamedThing"]
          }
        },
        "edges": {
          "e01": {
            "subject": "n0",
            "object": "n1"
          }
        }
      }
    }
  }
```
returns:
```
                        {
                            "attribute_type_id": "biolink:p_value",
                            "original_attribute_name": "pValue",
                            "value": 0.0019390149921821016
                        }
                    ],
                    "object": "EFO:0004308",
                    "predicate": "biolink:genetic_association",
                    "subject": "KEGG:hsa05322"
                }
```

**Comment by karafecho** [comment]
Thanks, Mark! This type of query could be very useful for Translator.

**Comment by karafecho** [comment]
Wow! Thanks for the update, Sarah.

**Comment by andrewsu** [comment]
@sstemann Can you put us in touch with the developers of the [RaMP API](https://rampdb.nih.gov/api)?  Since the API already exists, it's possible we can annotate/register it in SmartAPI and have it accessible through BTE without any further development.  We were about to get on that, but have one detail we'd like to follow up on first.

**Comment by sierra-moxon** [comment]
@andrewsu @sstemann - did either of these resources get incorporated into a Translator resource or should we keep this ticket open for further work?

**Comment by andrewsu** [comment]
Service Provider provides access to a rhea API -- more info in https://github.com/biothings/mychem.info/issues/37

RaMP is ongoing -- linking the currently open issue at https://github.com/biothings/biothings_explorer/issues/705

**Comment by codewarrior2000** [comment]
MolePro is in the process of assimilating the Reactome DB, which is "an open-source, open access, manually curated and peer-reviewed pathway database". 

In Reactome, the metabolic pathways are grouped under Metabolism, which has the Reactome identifier [R-HSA-1430728](https://reactome.org/content/detail/R-HSA-1430728).

**Comment by sstemann** [comment]
we deferred to BTEs RaMP integration.  If any of these are targeted for this funding phase of the program can folks please add a release (guppy, hammerhead, etc) to the label? @codewarrior2000 @andrewsu

**Comment by Mathelab** [comment]
@andrewsu  I agree that it makes more sense to improve upon the existing resources.  A couple of nuances worth mentioning:
- RaMP does 'harmonize' metabolite names across the resources, which is actually very difficult, particularly for lipids.  
- Biochemical pathways and chemistry seem to be pulled into Translator from other sources listed above so those may not be useful.  Not sure whether Classyfire is being pulled in but that ontology is very widely used in the metabo community.
- Seems like you could pull from LIPID MAPS directly or through RaMP to take advantage of the 'harmonization'.
- RaMP pulls in ontologies from HMDB not sure whether those would be useful.

Happy to discuss more during the relay as well.

**Comment by sstemann** [comment]
@andrewsu @Mathelab were there any changes we should look for in Hammerhead for this?

**Comment by andrewsu** [comment]
Thanks for the ping on this. Unfortunately, nothing new is planned on our end for Hammerhead. I've updated the tag to "Next Phase".  I've also renamed the issue to refine its scope, since I think the broader issue of "metabolic pathway data" is mostly solved (per [my comment above](https://github.com/NCATSTranslator/Feedback/issues/64#issuecomment-2330379535)).
